examples/05_stable_diffusion/scripts/compile.py

```diff
@@ -44,7 +45,9 @@ def compile_diffusers(
 ):
     logging.getLogger().setLevel(logging.INFO)
     torch.manual_seed(4896)
-
+    
+    local_dir = get_work_dir_location_diffusers()
```

Comment:
There is also an internal question about `--local-dir` option:

> Do we need to consider removing the option similar to here? What if user set it here but get override in get_work_dir_location_diffusers() ?

cc @frank-wei

Reply:
Addressed both the comments.